tools/cosmovisor/args.go

```diff
@@ -11,9 +11,10 @@ import (
 	"time"
 
 	cverrors "cosmossdk.io/tools/cosmovisor/errors"
+	"github.com/rs/zerolog"
```

Comment:
are these kind of changes okay @julienrbrt ? these imports seem to be grouping while saving

Reply:
Yeah, there is no logic change in cosmovisor then, so it's good :)

Reply:
nvm, I reverted those :)